jest.config.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    preset: './jest-preset.js',
+    snapshotSerializers: [
+        'jest-serializer-path',
+    ],
+};
```

Comment:
Should we have this file in our project? It isn't being used and won't be used when importing.

Reply:
This makes sense to solve absolute paths to relative paths, to prevent having /Users/ivo/... in snapshots.

Reply:
This makes it so pathing works in CI, same as with [`next-common-files`](https://github.com/moxystudio/next-common-files/blob/master/jest.config.js).

It isn't currently being bundled in the npm package.

Reply:
In talking about the file `jest.config.js`, not about the `jest-serializer-path`.